### ISSUE TITLE
Warning of invalid react native sdk version

### ISSUE THREAD

**Issue by thaingo** [infinitered/ignite#1955]
What's going on?

When launching a generated app, the following message shown:
`Warning: https://github.com/expo/react-native/archive/sdk-43.0.0.tar.gz is not a valid version. Version must be in the form of sdk-x.y.z. Please update your package.json file.`

---

Steps to reproduce

1. npx ignite-cli new Sample --expo --bundle=com.sample
2. cd Sample
5. yarn start
6. open iOS Simulator using `i`

---

`npx ignite-cli doctor` results:
```
System
  platform           darwin                                             
  arch               arm64                                              
  cpu                8 cores      Apple M1                              
  directory          ignite       /Users/thai.ngo/Documents/mine/ignite 

JavaScript (and globally-installed packages)
  node               16.15.0      /usr/local/bin/node 
  npm                8.5.5        /usr/local/bin/npm  
    corepack         0.10.0                           
    expo-cli         5.4.7                            
    ignite-cli       7.10.10                          
    npm              8.5.5                            
    yarn             1.22.18                          
  yarn               1.22.18      /usr/local/bin/yarn 
  pnpm               -            not installed       

Ignite
  ignite-cli         7.10.10      /usr/local/bin/ignite                        
  ignite src         build        /usr/local/lib/node_modules/ignite-cli/build 

Android
  java               17.0.3.1     /usr/bin/java 
  android home       -            undefined     

iOS
  xcode              13.4       
  cocoapods          1.11.3       /usr/local/bin/pod 
```

**Comment by isaaclyman** [comment]
I'm getting the warning as well. It's preventing JavaScript debugging from working in the iOS simulator.

**Comment by jamonholmgren** [comment]
This will be fixed with Ignite v8 (code-named Maverick). Sorry about the delay, we're working hard on it!

**Comment by frankcalise** [comment]
@isaaclyman @jir-f @thaingo 

For now, you can easily upgrade the Expo version via:

`expo-cli upgrade` and follow the prompts to version 46
then
`yarn start` or `yarn expo start`

Let me know if you have any issues, I just tested this with that latest version of ignite, 7.14.2

**Comment by frankcalise** [comment]
Resolved in #2003, will be released in [Ignite v8: Maverick](https://github.com/infinitered/ignite/milestone/19)
